### PR TITLE
Docs: console doc fixes WIP

### PR DESCRIPTION
**Pull request by jenmalloy**
### Type of change
- Documentation

### Description
Console docs updates to reflect new console UI

### Checklist
- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [X] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

**Comment by jenmalloy** [comment]
@k-wall, Could you please review for technical accuracy, and also add the new console screenshot?  Thanks!

File: documentation/modules/ref-view-message-conn-stats-table.adoc [hunk 1]
```diff
@@ -3,36 +3,36 @@
...
-|Standard address space only: Total number of messages sent as long the connection has existed |*Messages Out*
-|Total number of messages delivered |For the desired connection, expand the twisty on the left to show the *Senders* and *Receivers* tables; see the *Deliveries* columns.
-|Standard address space only: Username used by the client to connect |The third column
+|To view... |On the Connections page...
+|Messages received per second (computed over the last 5 minutes) |See *Messages In*
+|Standard address space only: Messages sent per second (computed over the last 5 minutes) |See *Messages Out*
+|Total number of messages delivered |Click the desired host name to show the list of senders and receivers; see the *Deliveries* column.
+// |Standard address space only: Username used by the client to connect |See the third column
 |===
 NOTE: For the brokered address space only, on the Connections page, the number of senders is either `0` or `1`. As soon as one or more senders exist, `1` is displayed rather than reflecting the actual number of senders.
```
Comment (jenmalloy) [inline documentation/modules/ref-view-message-conn-stats-table.adoc]: @k-wall, Just want to confirm that this note is still accurate for the brokered address space

File: documentation/modules/proc-change-address-space-auth-service-console.adoc [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// assembly-managing-address-spaces.adoc
+
+[id='proc-change-address-space-auth-service-console-{context}']
+= Changing the authentication service associated with an address space using the {ConsoleName}
+
+You can change the authentication service that is associated with an address space using the  {ConsoleName}.
```
Comment (k-wall) [inline documentation/modules/proc-change-address-space-auth-service-console.adoc]:
```suggestion
You can change the authentication service that is associated with an address space using the {ConsoleName}.
```

File: documentation/modules/proc-change-address-space-auth-service-console.adoc [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+.Prerequisites
+* You must have already created an address space. For more information see link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#proc-create-address-space-console-messaging[Creating an address space using the {ConsoleName}].
+
+.Procedure
+
+. Log in to the {ConsoleName}. For more information, see link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#logging-into-console-messaging[Accessing the {ConsoleName}].
+
+. Locate the address space for which you want to change the address space plan.
```
Comment (k-wall) [inline documentation/modules/proc-change-address-space-auth-service-console.adoc]:
```suggestion
. Locate the address space for which you want to change the authentication service.
```

**Comment by k-wall** [comment]
@jenmalloy sorry I intended to submit a PR against yours, but pushed to it by mistake,

I updated the existing screenshot.  I also include screenshots of the address/conns - perhaps it is worth including these into `ref-view-message-connection-stats-table`

**Comment by jenmalloy** [comment]
@k-wall, looks good to me. I think you will need to approve so I can merge?

File: documentation/modules/proc-logging-in-console.adoc [hunk 4]
```diff
@@ -6,7 +6,21 @@
...
+----
+
+* On OpenShift Container Platform 4.x, obtain the host name for the {ConsoleName} by running the following command:
+----
+oc get consolelink -l app=enmasse -o jsonpath={.spec.href}
+----
+endif::[]
+
+ifeval::["{cmdcli}" == "kubectl"]
+* ??
```
Comment (jenmalloy) [inline documentation/modules/proc-logging-in-console.adoc]: @k-wall Just occurred to me that we also need to include the command for Kubernetes access as well, correct?

**Pull request merged**